estate/models/estate_property.py

```diff
@@ -0,0 +1,24 @@
+from odoo import models, fields
+
+class EstatePropertyModel(models.Model):
```

Comment:
```suggestion
from odoo import models, fields

class EstateProperty(models.Model):
```
Class name will be just capitalized of what is given in `_name`